src/avutil.ml

```diff
@@ -26,12 +26,24 @@ let () =
 module Pixel_format = struct
   type t = Pixel_format.t
 
-  external bits : t -> int = "ocaml_avutil_bits_per_pixel"
+  external bits : t -> int = "ocaml_avutil_pixelformat_bits_per_pixel"
+  external planes : t -> int = "ocaml_avutil_pixelformat_planes"
+  external to_string : t -> string = "ocaml_avutil_pixelformat_to_string"
+  external of_string : string -> t = "ocaml_avutil_pixelformat_of_string"
 
   let bits (*?(padding=true)*) p =
     let n = bits p in
     (* TODO: when padding is true we should add the padding *)
     n
+
+  let planes fmt =
```

Comment:
These intermediate functions are not necessary, the external functions can be called directly. Is there a particular reason for keeping them?

Reply:
Not really, I've removed those.